docs/rootless.md

```diff
@@ -0,0 +1,41 @@
+# Running containerd as a non-root user
+
+A non-root user can execute containerd by using [`user_namespaces(7)`](http://man7.org/linux/man-pages/man7/user_namespaces.7.html).
+
+For example [RootlessKit](https://github.com/rootless-containers/rootlesskit) can be used for setting up a user namespace (along with mount namespace and optionally network namespace). Please refer to RootlessKit documentation for further information.
+
+## Daemon
+
+```console
+$ rootlesskit --net=slirp4netns --copy-up=/etc \
+  --state-dir=/run/user/1001/rootlesskit-containerd \
+  containerd -c config.toml
+```
+
+* `--net=slirp4netns --copy-up=/etc` is only required when you want to unshare network namespaces
+* Depending on the containerd plugin configuration, you may also need to add more `--copy-up` options, e.g. `--copy-up=/run`, which mounts a writable tmpfs on `/run`, with symbolic links to the files under the `/run` on the parent namespace.
```

Comment:
An example of this is v2 runtime: 
https://github.com/containerd/containerd/issues/2767